terraform/performance/variables.tf

```diff
@@ -0,0 +1,94 @@
+variable "region" {
+  type    = string
+  default = "us-west-2"
+}
+
+variable "ec2_instance_type" {
+  type    = string
+  default = "t3a.xlarge"
+}
+
+variable "ssh_key_name" {
+  type    = string
+  default = "cwagent-integ-test-key"
+}
+
+variable "ami" {
+  type    = string
+  default = "cloudwatch-agent-integration-test-ubuntu*"
+}
+
+variable "ssh_key_value" {
+  type    = string
+  default = ""
+}
+
+variable "user" {
+  type    = string
+  default = ""
+}
+
+variable "install_agent" {
+  description = "go run ./install/install_agent.go deb or go run ./install/install_agent.go rpm"
+  type        = string
+  default     = ""
```

Comment:
This will increase more further if we include `darwin`. However, to answer your questions, we can use `go run ./install/install_agent.go rpm` since linux is a major case.